src/common/settings.js

```diff
@@ -26,6 +26,7 @@ var loadDefault = function(version) {
         showTrayIcon: false,
         trayIconTheme: '',
         disablewebsecurity: true,
+        autostart: false,
```

Comment:
Yes, that was what I originally tried to do, but ended up with the config. Will try to do the other way again.